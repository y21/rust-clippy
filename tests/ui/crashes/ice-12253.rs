#![warn(clippy::indexing_slicing)]

const C: [u32; 5] = [0; 5];

fn main() {
    #[allow(unconditional_panic, overflowing_literals, clippy::no_effect)]
    C[0xfffffe7fffffffffffffffffffffffff];
}
