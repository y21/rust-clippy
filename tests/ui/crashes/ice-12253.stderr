error: index is out of bounds
  --> $DIR/ice-12253.rs:7:5
   |
LL |     C[0xfffffe7fffffffffffffffffffffffff];
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[deny(clippy::out_of_bounds_indexing)]` on by default

error: aborting due to 1 previous error

